_posts/2021-01-10-boa-release-11.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title: "Boa release v0.11"
+author: Boa Developers
+---
+
+Boa has reached a new release. v0.11, our biggest one yet. Since v0.10 we've closed 77 issues, merged 129 pull requests. The engine has been faster and more compliant to the spec. Below are some of the highlights but please see the [changelog link] for more information.
+
+## Test 262
+
+Test262 is the implementation conformance test suite maintained by TC39. It's used by nearly all engines to measure how conformant they are to the specification. Boa pulls the tests in-tree and runs against PRs and on a nightly basis. Test262 doesn't sit still, so since the last release we've updated our reference to make sure we're running against the latest spec. You can find more about Test262 [here](https://github.com/tc39/test262).
```

Comment:
I don't think the term nightly fits here, what doyou think?
```suggestion
Test262 is the implementation conformance test suite maintained by TC39. It's used by nearly all engines to measure how conformant they are to the specification. Boa pulls the tests in-tree and runs them against all PRs. Test262 doesn't sit still, so since the last release we've updated our reference to make sure we're running against the latest spec. You can find more about Test262 [here](https://github.com/tc39/test262).
```